### PR TITLE
Integration and testing

### PR DESCRIPTION
**Pull request by robertecurtin**
- [x] Fixed submodule path
- [x] Tuned some values, still not done testing
- [x] Added blocking for line avoidance
- [x] Fix tests
- [x] Made modules rely on `tuning.h` for configurable values

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -44,4 +44,4 @@
       "preLaunchTask": "debug-deps-test"
     }
   ]
-}
+}
```
Comment (robertecurtin) [inline .vscode/launch.json]:
```suggestion
}

```

File: .vscode/tasks.json [hunk 2]
```diff
@@ -30,4 +30,4 @@
       }
     }
   ]
-}
+}
```
Comment (robertecurtin) [inline .vscode/tasks.json]:
```suggestion
}

```

File: .vscode/tasks.json [hunk 3]
```diff
@@ -19,7 +19,7 @@
     {
       "label": "debug-deps-test",
       "type": "shell",
-      "command": "make -j8 -f test.mk debug-deps",
+      "command": "",
```
Comment (ryanplusplus) [inline .vscode/tasks.json:22]: Why did you do this? This is what makes it build before debug.

File: src/application/tactics/tuning.h [hunk 4]
```diff
@@ -0,0 +1,21 @@
+#ifndef tuning_h
```
Comment (ryanplusplus) [inline src/application/tactics/tuning.h:6]:
```suggestion
/*!
 * @file
 * @brief
 */

#ifndef tuning_h
```

File: src/application/tactics/tuning.h [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+  charge_full_power = 100,
+  charge_reduced_power = 50
+};
+
+enum {
+  seeking_near_wheel_power = 40,
+  seeking_far_wheel_power = 100
+};
+
+#endif
```
Comment (ryanplusplus) [inline src/application/tactics/tuning.h]:
```suggestion
#endif

```

File: .vscode/launch.json (hunk 1 above)
Comment (ryanplusplus) [inline .vscode/launch.json]:
```suggestion
}

```

**Pull request merged**